src/components/Primary-Button/Button.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { PrimaryButton } from './Button';
+import { BUTTON_VARIANTS, BUTTON_SIZES, BUTTON_SHAPES } from './Button.constants';
+
+import type { ButtonVariant, ButtonSize, ButtonShape } from './Button.types'; // Import types correctly
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof PrimaryButton> = {
+  title: 'Components/PrimaryButton',
+  component: PrimaryButton,
+  parameters: {
+    layout: 'centered', // Centers the component in the Storybook panel
+    docs: {
+      description: {
+        component: 'Button with multiple variants, sizes, and shapes built on MUI.',
+      },
+    },
+  },
+  argTypes: {
+    variant: { control: 'select', options: Object.values(BUTTON_VARIANTS) as ButtonVariant[] }, // Use ButtonVariant type
+    size: { control: 'select', options: Object.values(BUTTON_SIZES) as ButtonSize[] }, // Use ButtonSize type
+    shape: { control: 'select', options: Object.values(BUTTON_SHAPES) as ButtonShape[] }, // Use ButtonShape type
+    loading: { control: 'boolean' },
+    disabled: { control: 'boolean' },
+    fullWidth: { control: 'boolean' },
+    children: { control: 'text' },
+    onClick: { action: 'clicked' },
+  },
+  args: {
+    children: 'Click me',
+    variant: 'primary',
+    size: 'medium',
+    shape: 'rounded',
+    loading: false,
+    disabled: false,
+    fullWidth: false,
+  },
+};
+
+export default meta;
```

Comment:
This file uses a default export for Storybook meta. The project convention states 'named exports' for components, but Storybook requires a default export for CSF meta. Please document or note the repository exception for story files so linting/rules don't conflict with Storybook requirement.

**Sugerencia:**
```typescript
Keep the default export for Storybook (it's required). Add a comment referencing the repo convention exception or update lint rules to allow default export for story meta files.
```